pkg/services/ngalert/api/tooling/definitions/provisioning_templates.go

```diff
@@ -5,20 +5,34 @@ package definitions
 // Get all message templates.
 //
 //     Responses:
-//       200: []MessageTemplate
+//       200: MessageTemplateResponse
```

Comment:
Why do we need to change this? Do our APIs only support returning a JSON object and not an array?

Reply:
they do but I think swagger does recognises this syntax therefore it should become:
https://github.com/grafana/grafana/blob/c031ba3a3cc88ad67090fb6de9a5a376a1d29af1/pkg/services/ngalert/api/tooling/definitions/provisioning_templates.go#L16

https://github.com/grafana/grafana/blob/c031ba3a3cc88ad67090fb6de9a5a376a1d29af1/pkg/services/ngalert/api/tooling/definitions/provisioning_templates.go#L31-L35